app/__init__.py

```diff
@@ -34,6 +31,16 @@ def create_app(config_name):
         search_api_client=search_api_client
     )
 
+    for framework_data in data_api_client.find_frameworks().get('frameworks'):
```

Comment:
Yeah I think it's peanuts in this day and age.  Doesn't mean we shouldn't think about it though.